test/test_gpu_transpose.cpp

```diff
@@ -1038,7 +1038,7 @@ TEST_F(NVFuserTest, FusionTransposeBankConflict9_CUDA) {
   auto tv1 = set(tv0);
   auto tv2 = transpose(tv1, 0, 1);
   auto tv3 = set(tv2);
-  fusion.addOutput(tv2);
```

Comment:
This is a bug fix, I don't think it makes any difference, but it is weird that we define a tv3 but never use it...